src/content/docs/en/guides/styling.mdx

```diff
@@ -485,6 +485,24 @@ npm install less
 
 Use `<style lang="less">` in `.astro` files.
 
+### LightningCSS
+
+```shell
+npm install lightningcss
+```
+
+Update your `astro.config.ts`
+
+```diff
++  vite: {
++    css: {
++      transformer: "lightningcss",
++    },
++  },
+```
+
+Then LightningCSS will optimize and down-level your vanilla CSS.
```

Comment:
``````suggestion
### LightningCSS

```shell
npm install lightningcss
```

Update your `vite` configuration in `astro.config.mjs`:

```js title="astro.config.mjs" ins={4-8}
import { defineConfig } from 'astro/config'

export default defineConfig({
  vite: {
    css: {
      transformer: "lightningcss",
    },
  },
})
  
```
``````
Just updated this for Expressive code!

Also, in [the Lightning CSS docs](https://lightningcss.dev/docs.html#with-vite) they show also setting a `build` option for Vite. Do we not have to do that because Astro does the building part?

Lastly, we don't really say what any of the other packages do, so I'd suggest not having any description for now. I can see that we also don't use our package manager tabs here either, so that means this page is probably do for a bit of an overhaul anyway, and maybe we can in fact add at least a link to each package and/or a one-liner to make this a little nicer. But for now, how is this?

Reply:
I agree with your assertions, I'm entirely uncertain about how much config is required. I think in the form I suggested it just does down-leveling and not minifying? And we rely on Astro for that? Adding the entire Vite guide might be helpful.